client.go

```diff
@@ -235,28 +239,33 @@ func (c *TunaSessionClient) OnConnect() chan struct{} {
 // RotateOne create a new tuna exit and replace the i-th one. New connections
 // accepted will use new tuna exit, existing connections will not be affected.
 func (c *TunaSessionClient) RotateOne(i int) error {
+	// Close old one first to avoid dialer get old one public address
```

Comment:
I don't think it's a good idea to close first. It's important to make rotation 0 down time.

Reply:
It happened that when one connection was closed, the new TunaExit was not ready, but Dialer sent a request for public addresses and got the old TunaExit public address. It seems it takes several seconds for new tunaExit to get ready.

Reply:
Restored to previous version